benefits/static/css/styles.css

```diff
@@ -86,8 +86,13 @@ footer {
   padding-bottom: 1rem;
 }
 
-footer .footer-links a {
+footer.global-footer .footer-links a {
   font-size: 16px;
+  color: #0590cd;
+}
+
+footer.global-footer .footer-links a:hover {
```

Comment:
This worked when I tested on Chrome in [device mode](https://developer.chrome.com/docs/devtools/device-mode/). 👍 

When I tested in Firefox in [Responsive Design Mode](https://firefox-source-docs.mozilla.org/devtools-user/responsive_design_mode/index.html) though, I didn't ever see the purple color. When I "tap" the link, it seems like the `:focus` state is taking effect, and the link turns an orange color which is coming from the CA Web Template styles. 

Here was the best screenshot I could take of it:
![image](https://user-images.githubusercontent.com/25497886/172452871-69f3a3d4-b482-422c-b377-a631cde43af9.png)

And here is simulating that state using devtools:
![image](https://user-images.githubusercontent.com/25497886/172453126-61c55608-43fb-4886-8e21-283bb99d4351.png)

I think to be safe, we could add a selector for the `:focus` state here?

Reply:
Great catch, thank you!

Added the `focus` and `active` selectors.